azure_license-server/main.tf

```diff
@@ -0,0 +1,90 @@
+variable "azure_region" {}
+variable "resource_name" {}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = var.resource_name
+  location = var.azure_region
+}
+
+resource "azurerm_virtual_network" "vnet" {
+  name                = "sentieon-vnet"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  address_space       = ["172.31.0.0/16"]
+}
+
+resource "azurerm_subnet" "subnet" {
+  name                 = "sentieon-subnet"
+  virtual_network_name = azurerm_virtual_network.vnet.name
+  resource_group_name  = azurerm_resource_group.rg.name
+  address_prefixes     = ["172.31.0.0/24"]
+}
+
+
+resource "azurerm_public_ip" "license_server_public_ip" {
+  name                = "license-server-public-ip"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  allocation_method   = "Dynamic"
+}
+
+resource "azurerm_network_interface" "license_server_nic" {
+  name                      = "license-server-nic"
+  location                  = azurerm_resource_group.rg.location
+  resource_group_name       = azurerm_resource_group.rg.name
+
+  ip_configuration {
+    name                          = "ipconfig1"
+    subnet_id                     = azurerm_subnet.subnet.id
+    private_ip_address_allocation = "Dynamic"
+    public_ip_address_id          = azurerm_public_ip.license_server_public_ip.id
+  }
+}
+
+resource "tls_private_key" "ssh" {
```

Comment:
Can we also output the `private_key_pem` to a file? This will allow users to avoid manually reading the .tfstate file before ssh-ing into the vm.